docs/examples.md

```diff
@@ -3,44 +3,45 @@ title: Examples
+- [Subscriptions (simple)](../examples/simple-subscriptions)
+- [Subscriptions (using Redis)](../examples/redis-subscriptions)
+- [Interfaces](../examples/interfaces-inheritance)
+- [Extensions (metadata)](../examples/extensions)
 
 ## Features usage
 
-- [Dependency injection (IoC container)](https://github.com/MichalLytek/type-graphql/tree/master/examples/using-container)
-  - [Scoped containers](https://github.com/MichalLytek/type-graphql/tree/master/examples/using-scoped-container)
-- [Authorization](https://github.com/MichalLytek/type-graphql/tree/master/examples/authorization)
-- [Validation](https://github.com/MichalLytek/type-graphql/tree/master/examples/automatic-validation)
-  - [Custom validation](https://github.com/MichalLytek/type-graphql/tree/master/examples/custom-validation)
-- [Types inheritance](https://github.com/MichalLytek/type-graphql/tree/master/examples/interfaces-inheritance)
-- [Resolvers inheritance](https://github.com/MichalLytek/type-graphql/tree/master/examples/resolvers-inheritance)
-- [Generic types](https://github.com/MichalLytek/type-graphql/tree/master/examples/generic-types)
-- [Mixin classes](https://github.com/MichalLytek/type-graphql/tree/master/examples/mixin-classes)
-- [Middlewares and Custom Decorators](https://github.com/MichalLytek/type-graphql/tree/master/examples/middlewares-custom-decorators)
-- [Query complexity](https://github.com/MichalLytek/type-graphql/tree/master/examples/query-complexity)
+- [Dependency injection (IoC container)](../examples/using-container)
+  - [Scoped containers](../examples/using-scoped-container)
+- [Authorization](../examples/authorization)
+- [Validation](../examples/automatic-validation)
+  - [Custom validation](../examples/custom-validation)
+- [Types inheritance](../examples/interfaces-inheritance)
+- [Resolvers inheritance](../examples/resolvers-inheritance)
+- [Generic types](../examples/generic-types)
+- [Mixin classes](../examples/mixin-classes)
+- [Middlewares and Custom Decorators](../examples/middlewares-custom-decorators)
+- [Query complexity](../examples/query-complexity)
 
 ## 3rd party libs integration
 
-- [TypeORM (manual, synchronous) \*](https://github.com/MichalLytek/type-graphql/tree/master/examples/typeorm-basic-usage)
-- [TypeORM (automatic, lazy relations) \*](https://github.com/MichalLytek/type-graphql/tree/master/examples/typeorm-lazy-relations)
-- [MikroORM](https://github.com/MichalLytek/type-graphql/tree/master/examples/mikro-orm)
-- [Typegoose](https://github.com/MichalLytek/type-graphql/tree/master/examples/typegoose)
-- [Apollo federation](https://github.com/MichalLytek/type-graphql/tree/master/examples/apollo-federation)
-- [Apollo Cache Control](https://github.com/MichalLytek/type-graphql/tree/master/examples/apollo-cache)
-- [Apollo Client local state](https://github.com/MichalLytek/type-graphql/tree/master/examples/apollo-client)
+- [TypeORM (manual, synchronous) \*](../examples/typeorm-basic-usage)
```

Comment:
I wonder how does it work on the website (Docusaurus)? How does it know to redirect into github instead of looking into docs? I think you went too far with the refactor of links

Reply:
![image](https://user-images.githubusercontent.com/10618781/220898008-bb8d994e-698e-4cf4-b38f-e521d1b0cc66.png)

@carlocorradini Please revert all the changes to examples on github, they have to be absolute url.
I would be great if you could write a simple script that will update the urls from `master` to tags on relase - see `versioned_docs` 🙏